### ISSUE TITLE
Can't override CAS_AUTH_CLASS

### ISSUE THREAD

**Issue by vauxia** [nitmir/django-cas-server#17]
I would like to use CAS with an LDAP authentication backend. I added the following line to my app's config/settings/common.py:

`CAS_AUTH_CLASS = 'cas_server.auth.LdapAuthUser'
`

When I POST the login form, I get the following error message:

```
TypeError at /cas/login
'unicode' object is not callable
...
Exception Value:    
'unicode' object is not callable
Exception Location: /python2.7/site-packages/cas_server/forms.py in clean, line 126
```

Line 126 reads of forms.py reads:

`auth = utils.import_attr(settings.CAS_AUTH_CLASS)(cleaned_data["username"]) 
`

It's having a problem instantiating the Authentication class defined via CAS_AUTH_CLASS. So I copied the value from cas_server/default_settings.py:

`CAS_AUTH_CLASS = 'cas_server.auth.DjangoAuthUser'`

_But I got the same TypeError_.

The TypeError does NOT occur when I comment out CAS_AUTH_CLASS in my settings/common.py. Form submission works as expected, but of course it doesn't use LDAP.

So there's something going on with overriding default_settings in settings/common.py. It may be user error, because Django is pretty new for me.

**Comment by nitmir** [comment]
Which version of python and django are you using ?

**Comment by vauxia** [comment]
Django 1.10 and python 2.7

**Comment by nitmir** [comment]
I managed to reproduce. It happen in python2 then `CAS_AUTH_CLASS` is a `unicode` and not a `str`.

So having `CAS_AUTH_CLASS = u"cas_server.auth.DjangoAuthUser"` will crash, and `CAS_AUTH_CLASS = "cas_server.auth.DjangoAuthUser"` will not.

It should be fixed by the commit 37c975eaf7b3c422fb1b21ead1db7ac1d46c7c88, and both forms should work in the next version.

**Comment by nitmir** [comment]
Fixed by #22
